src/main/java/bleuauction/bleuauction_be/server/notice/controller/NoticeController.java

```diff
@@ -36,107 +30,57 @@
 @Slf4j
 @RestController
 @RequiredArgsConstructor
+@RequestMapping("/api/notice")
 public class NoticeController {
 
   private final NoticeService noticeService;
-  private final EntityManager entityManager;
   private final MemberService memberService;
-  private final NoticeRepository noticeRepository;
-  private final MemberRepository memberRepository;
-  private final PasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
   private final CreateJwt createJwt;
-  private final NcpObjectStorageService ncpObjectStorageService;
   private final AttachService attachService;
 
 
-  //등록
-  @GetMapping("/api/notice/new")
-  public NoticeForm createForm() {
-    NoticeForm noticeForm = new NoticeForm();
-    return noticeForm;
-  }
 
-//
   // 등록 처리(관리자 회원)
-  @PostMapping("/api/notice/new")
+  @PostMapping("/new")
   @Transactional
-  public ResponseEntity<?>  notice(@RequestHeader("Authorization") String  authorizationHeader, Notice notice, @RequestParam(name = "multipartFiles",required = false) List<MultipartFile> multipartFiles) {
+  public ResponseEntity<?>  notice(@RequestHeader("Authorization") String  authorizationHeader, Notice notice, @RequestParam(name = "multipartFiles",required = false) List<MultipartFile> multipartFiles) throws Exception{
 
     createJwt.verifyAccessToken(authorizationHeader);
     TokenMember tokenMember = createJwt.getTokenMember(authorizationHeader);
     Optional<Member> loginUser = memberService.findByMemberNo(tokenMember.getMemberNo());
 
-    if(loginUser.get().getMemberCategory() == A) {
-    notice.setMemberNo(loginUser.get());
-
-    if (multipartFiles != null && multipartFiles.size() > 0) {
-      ArrayList<Attach> attaches = new ArrayList<>();
-      for (MultipartFile multipartFile : multipartFiles) {
-        if (multipartFile.getSize() > 0) {
-          Attach attach = ncpObjectStorageService.uploadFile(new Attach(),
-                  "bleuauction-bucket", "notice/", multipartFile);
-          notice.addNoticeAttach(attach);
-        }
-      }
-    }
-
-    notice = entityManager.merge(notice);
-    noticeService.enroll(notice);
-
+    noticeService.enroll(notice,multipartFiles,loginUser.get());
     log.info("notice/postnew");
 
-    return ResponseEntity.status(HttpStatus.CREATED).body("Notice created successfully");}
-    else {
+    try {
+      return ResponseEntity.status(HttpStatus.CREATED).body("Notice created successfully");
+    } catch (Exception e) {
       return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("관리자 권한이 필요합니다");
     }
   }
```

Comment:
Service를 보고오니 Catch문은 꼭 필요하진않네여
이렇게면 끝나지 않을까 싶어요
```suggestion
      return ResponseEntity.status(HttpStatus.CREATED).body("Notice created successfully");
  }
```